services/billing.py

```diff
@@ -381,7 +378,6 @@ def create_checkout_session(self, owner: Owner, desired_plan):
             success_url=success_url,
             cancel_url=cancel_url,
             customer=customer,
```

Comment:
I think you could write this as `customer = owner.stripe_customer_id`, as it will be None or the value, so 369-371 isn't needed

Reply:
oh yeah good point! will make the change